### ISSUE TITLE
Allos maintain local sender blacklist

### ISSUE THREAD

**Issue by rimas-kudelis** [vexim/vexim2#231]
While trying to teach my spamassassin that the newsletters I don't remember subscribing to are in fact spam, I figured out that (being mailing lists) they probably come from same server(s), so instead, why not simply tell Exim to not accept any e-mails coming from these servers. It would be nice to have an example of a black list in our config files.

It seems that in Debian, this is already possible with the `local_host_blacklist` file, but we could probably copy that mechanism into our non-Debian files as well.

As for v3, the list could probably go into the database instead of being stored in a plain text file.

**Comment by Udera** [comment]
I normally prefer a maintained blacklist for such issues. Maintaining a manual list for yourself takes a lot of time and is only worth it for very annoying mail servers. We should give the admin the possibilty to black or whitelist a specific server.

**Comment by rimas-kudelis** [comment]
Yes, I was talking from the admin perspective here. I don't think many normal users would use this feature, even if available.
On the other hand, we could probably make it possible to submit suggestions to the list for the siteadmin to review.
